test/test_log.py

```diff
@@ -48,7 +49,7 @@ def tearDown(self):
         logging.getLogger().handlers.clear()
 
     def test_default_logging(self):
-        ops.log.setup_root_logging(self.backend)
+        ops.log.setup_root_logging(self.backend)  # type: ignore
```

Comment:
Hmm, yeah, this is a bit annoying. I suppose ideally `setup_root_logging` and the like should take a `typing.Protocol`, but that's it's own messiness.

Could we just make `FakeModelBackend` inherit from `_ModelBackend` and remove these type-ignores?

Reply:
> Hmm, yeah, this is a bit annoying. I suppose ideally `setup_root_logging` and the like should take a `typing.Protocol`, but that's it's own messiness.

Yeah, it seemed like quite a bit of code for little gain.

> Could we just make `FakeModelBackend` inherit from `_ModelBackend` and remove these type-ignores?

Sure. I wondered if you'd rather keep the stub minimal, but happy to go with this.

Reply:
> keep the stub minimal

Yeah, I wondered that, but if any of the other methods are called they'll fail hard because the hook tool is missing, so I think this is fine. Thanks.